seed/main/hi_Latn.xml

```diff
@@ -1006,15 +2104,2224 @@ annotations.
+				</long>
+			</metazone>		
+			<metazone type="Amazon">
+				<long>	
+					<generic>Amazon Time</generic>
+					<standard>Amazon Standard Time</standard>
+					<daylight>Amazon Summer Time</daylight>
+				</long>
+			</metazone>		
+			<metazone type="America_Central">
+				<long>	
+					<generic>North America Central Time</generic>
+					<standard>North America Central Standard Time</standard>
+					<daylight>North America Central Daylight Time</daylight>
+				</long>
+			</metazone>		
+			<metazone type="America_Eastern">
+				<long>	
+					<generic>North America Eastern Time</generic>
+					<standard>North America Eastern Standard Time</standard>
+					<daylight>North America Eastern Daylight Time</daylight>
+				</long>
+			</metazone>		
+			<metazone type="America_Mountain">
+				<long>	
+					<generic>North America Mountain Time</generic>
+					<standard>North America Mountain Standard Time</standard>
+					<daylight>North America Mountain Daylight Time</daylight>
+				</long>
+			</metazone>		
+			<metazone type="America_Pacific">
+				<long>	
+					<generic>North America Pacific Time</generic>
+					<standard>North America Pacific Standard Time</standard>
+					<daylight>North America Pacific Daylight Time</daylight>
+				</long>	
+			</metazone>		
+			<metazone type="Anadyr">
+				<long>	
+					<generic>Anadyr Time</generic>
+					<standard>Anadyr Standard Time</standard>
+					<daylight>Anadyr Summer Time</daylight>
+				</long>
+			</metazone>		
+			<metazone type="Apia">
+				<long>	
+					<generic>Apia Time</generic>
+					<standard>Apia Standard Time</standard>
+					<daylight>Apia Daylight Time</daylight>
+				</long>
+			</metazone>		
+			<metazone type="Arabian">
+				<long>	
+					<generic>Arabian Time</generic>
+					<standard>Arabian Standard Time</standard>
+					<daylight>Arabian Daylight Time</daylight>
+				</long>
+			</metazone>		
+			<metazone type="Argentina">
+				<long>	
+					<generic>Argentina Time</generic>
+					<standard>Argentina Standard Time</standard>
+					<daylight>Argentina Summer Time</daylight>
+				</long>
+			</metazone>		
+			<metazone type="Argentina_Western">
+				<long>	
+					<generic>Western Argentina Time</generic>
+					<standard>Western Argentina Standard Time</standard>
+					<daylight>Western Argentina Summer Time</daylight>
+				</long>
+			</metazone>		
+			<metazone type="Armenia">
+				<long>	
+					<generic>Western Argentina Time</generic>
+					<standard>Western Argentina Standard Time</standard>
+					<daylight>Western Argentina Summer Time</daylight>
+				</long>
+			</metazone>		
+			<metazone type="Atlantic">
+				<long>	
+					<generic>Western Argentina Time</generic>
+					<generic>Atlantic Time</generic>
+					<standard>Western Argentina Standard Time</standard>
+					<standard>Atlantic Standard Time</standard>
+					<daylight>Western Argentina Summer Time</daylight>
+					<daylight>Atlantic Daylight Time</daylight>
+				</long>
+			</metazone>		
```

Comment:
```suggestion
			<metazone type="Atlantic">
				<long>	
					<generic>Atlantic Time</generic>
					<standard>Atlantic Standard Time</standard>
					<daylight>Atlantic Daylight Time</daylight>
				</long>
			</metazone>		
```

This is the actual error. But the other duplications are problematic also.

Reply:
Fixed! Thank you!